features/filmStrip/components/FilmStrip.js

```diff
@@ -40,6 +40,10 @@ class FilmStrip extends Component {
                                     participant={p}
                                     videoTrack={videoTrack}
                                     audioTrack={audioTrack}
+                                    audioMuted={!audioTrack ||
```

Comment:
We already have audioTrack and videoTrack inside "VideoThumbnail" component, so we can do "isMuted()" checks internally and don't pass them from outside. WDYT?